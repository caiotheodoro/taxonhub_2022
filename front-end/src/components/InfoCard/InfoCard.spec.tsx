import { render } from "@testing-library/react";
import { FiHome } from "react-icons/fi";
import { InfoCard } from "./index";


describe("NavItem components", () => {
  it("should render correctly", () => {
    const { container } = render(<InfoCard  />);

    expect(container).toMatchSnapshot();
  });

 
});
