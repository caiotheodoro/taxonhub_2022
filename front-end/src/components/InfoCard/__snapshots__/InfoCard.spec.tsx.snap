// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NavItem components should render correctly 1`] = `
<div>
  <div
    class="css-16aiygp"
  >
    Formato do header CSV:
    <div
      class="css-1suc48s"
    >
      <div
        class="chakra-table__container css-zipzvv"
      />
      <p
        class="chakra-text css-xvczcd"
      />
      <p
        class="chakra-text css-1pmhnzl"
      >
        BinomialName
      </p>
    </div>
  </div>
</div>
`;
